client/src/components/IngredientShow.js

```diff
@@ -29,7 +29,7 @@ const IngredientShow = (props) => {
   return(
     <div>
       <h2>{capName}</h2>
-      <h4>Measurement: {ingredient.measurement} {ingredient.unit}</h4>
+      <h4>Measurement: {Number(ingredient.amount)} {ingredient.unit}</h4>
```

Comment:
I dont know that I now understand the purpose of this page. Its fine to still have it, but you might remove it till later. Why would we need to see an Ingredient show page?

And the measurement unit and amount...that would only be for one specific user's stock of the ingredient, whereas this data suggest that you are just grabbing some random measurement for this ingredient...?